dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java

```diff
@@ -201,48 +201,4 @@ public void testArtifactToClasspath_notToGenerateRepositoryException()
     List<Path> paths = ClassPathBuilder.artifactsToClasspath(ImmutableList.of(jamonApiArtifact));
     Truth.assertThat(paths).isNotEmpty();
   }
-
-  @Test
-  public void testArtifactToClasspath_reportAggregatedRepositoryException()
-      throws RepositoryException {
-    // jamon has transitive dependency to jmxtools, which does not exist in Maven central
```

Comment:
With the change, resolving `com.jamonapi:jamon:2.81` no longer throws an exception.